Content.Server/Chemistry/EntitySystems/SolutionGasHeatConductivitySystem.cs

```diff
@@ -0,0 +1,45 @@
+using Content.Server.Atmos.EntitySystems;
+using Content.Server.Chemistry.Components.SolutionManager;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.Chemistry.EntitySystems;
+
+public sealed class SolutionGasHeatConductivitySystem : EntitySystem
+{
+    [Dependency] private readonly AtmosphereSystem _atmosphereSystem = default!;
+    [Dependency] private readonly SolutionContainerSystem _solutionContainerSystem = default!;
+    [Dependency] private readonly TransformSystem _transformSystem = default!;
+
+    /// <inheritdoc/>
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        var query = EntityQueryEnumerator<SolutionContainerManagerComponent, SolutionGasHeatConductivityComponent>();
+        while (query.MoveNext(out var uid, out var solutionContainer, out var solutionGasHeatConductivity))
+        {
+            if (!_solutionContainerSystem.TryGetSolution(uid, solutionGasHeatConductivity.Solution, out var solution) || solution.Volume == 0)
+                continue;
+
+            var tf = Transform(uid);
+            var grid = tf.GridUid;
+            var map = tf.MapUid;
+            var indices = _transformSystem.GetGridOrMapTilePosition(uid, tf);
+            var mixture = _atmosphereSystem.GetTileMixture(grid, map, indices);
```

Comment:
can this not just use AtmosExposed instead rather than doing it's own query?

Reply:
I did not know that was a thing, fixed in fe6759f586f961ae3a04e0b7132aff89d8206c0b. I had to make `AtmosphereSystem.ExposedUpdateDelay` public since I relied on `frameTime` before. I hope that's fine.